replications/internal/queue_management.go

```diff
@@ -11,18 +12,23 @@ import (
 	"github.com/influxdata/influxdb/v2/kit/platform"
 	"github.com/influxdata/influxdb/v2/pkg/durablequeue"
 	"github.com/influxdata/influxdb/v2/replications/metrics"
+	"github.com/influxdata/influxdb/v2/replications/remotewrite"
 	"go.uber.org/zap"
 )
 
+type remoteWriter interface {
+	Write(context.Context, []byte) error
```

Comment:
Is there a case where the context here wouldn't just be `context.Background()`?

Reply:
I think we might need to propagate a context through here from the queue manager for https://github.com/influxdata/influxdb/issues/22944. It's mostly here as a placeholder for now, I'll add a TODO in the code to either make use of it or remove it if it doesn't end up being needed.